USAGE.md

```diff
@@ -249,15 +249,19 @@ $ aws lambda invoke --function-name import-status \
 
 ## Receive Import Status updates by subscribing to our AWS SNS Topic
 
-The ARN of our SNS Topic is `arn:aws:sns:ap-southeast-2:715898075157:geostore-import-status` which
-you may choose to subscribe to.
+The ARN of our SNS Topic is `arn:aws:sns:ap-southeast-2:<YOUR_ACCOUNT_ID>:geostore-import-status`
+which you may choose to subscribe to.
 
 You may also choose to apply a subscription filter policy, which will filter notifications for a
 specific dataset or specific statuses. Included in the example is all the valid statuses.
 
+The Geostore will store a dataset in a top level directory name with format of (dataset
+title)-(dataset id) You can filter SNS topics for a dataset by providing the entire dataset
+directory name or by providing just the title in a 'prefix' object. Examples below.
+
 ```json
 {
-  "dataset_id": ["01F9ZGZ4EGEZ9R4Z48VNQHKB0Z", "01F9ZFRK12V0WFXJ94S0DHCP65"],
+  "dataset_id": [{ "prefix": "Taranaki" }, "Taranaki_2020-01F9ZFRK12V0WFXJ94S0DHCP65"],
```

Comment:
What's going on here? Are we actually returning a `List[Union[Dict[str, str], str]]`?

Reply:
Also a little confused. I would have thought it would be something like
`"dataset_id": [{ "prefix": "Taranaki" }]`

Reply:
We're  not returning anything. This is an example json policy that can be used to filter the SNS messages - it has one example where a prefix is used one where the whole string is matched.